cmd/capabilities_fixes.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+
+func fixCapabilitiesNIL(resource k8sRuntime.Object) k8sRuntime.Object {
```

Comment:
Ditto here. Should we consider getting rid of `ErrorCapabilitiesNIL` all together and instead just throw a bunch of  `ErrorCapabilityNotDropped`s?